conda.recipe/meta.yaml

```diff
@@ -14,58 +14,54 @@ requirements:
     - python
     - distribute
     - setuptools
+    - nodejs >=4.1
 
   run:
     # bokeh
     - python
-    - numpy
-    - pandas
     - six
     - requests
-    - dateutil
     - pyyaml
-    - pyzmq
-    - tornado >=4.0
+    - python-dateutil
+    - jinja2
+    - numpy
+    - pandas
 
     # server
     - flask
-    - werkzeug
-    - greenlet
-    # - redis                [linux or osx]
-    # - redis-py             [linux or osx]
-    # - abstract-rendering   [py2k]
-
-    # CLI
-    # - click
+    - pyzmq
+    - tornado >=4.0
```

Comment:
In future, we should really investigate conda features (or whatever this is called). Like in gentoo, it would be great just install basic dependencies and all other under a server flag.